res/CircleSchema/0.8/circle_schema.fbs

```diff
@@ -619,6 +620,7 @@ union BuiltinOptions {
   BitcastOptions,
   BitwiseXorOptions,
   RightShiftOptions,
+  RoPEOptions = 250,
```

Comment:
i will change op number after rmsnorm is added because the number overlaps with rmsnorm